common/js/src/container-helpers.js

```diff
@@ -0,0 +1,43 @@
+/** @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.provide('GoogleSmartCard.ContainerHelpers');
+
+goog.require('GoogleSmartCard.Logging');
+goog.require('goog.array');
+goog.require('goog.object');
+
+goog.scope(function() {
+
+/** @const */
+var GSC = GoogleSmartCard;
+
+/**
+ * Converts the given Map to an Object.
+ * @param {!Map} map
+ * @return {!Object}
+ */
+GSC.ContainerHelpers.buildObjectFromMap = function(map) {
```

Comment:
What (should happen or) happens if the Map has a key that is not a string? Should we test that?

Reply:
Good point, added an assertion (I think it's reasonable to assume only strings and numbers to be passed here by a valid code) and tests.